src/backend/common/CommandBufferStateTracker.cpp

```diff
@@ -164,10 +164,6 @@ namespace backend {
     };
```

Comment:
Actually currently most things are gated on RENDER_PIPELINE, we don't have a RENDERPASS validation aspect. I'm not sure we need a SUBPASS validation aspect (at least, not right at the moment) - I'm not spotting any area where the current validation is wrong.